### PR TITLE
Update Beatmap Nominators List

### PR DESCRIPTION
**Pull request by celerih**
Still waiting on a few ppl to give me their Additional Languages, but this adds the new wave of beatmap nominators, as well as removing the resignations and removals of the past month or so

### CODE REVIEW

**Comment by celerih** [comment]
> should probably update the translations too since they aren't far behind. you can just copy/paste the table contents to each other article but leave the table headers as is (and keep `outdated: true` because I think that refers to the content before the tables)

Done!

File: wiki/People/Beatmap_Nominators/en.md [hunk 1]
```diff
@@ -150,13 +167,13 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
 | [Ascendance](https://osu.ppy.sh/users/2931883)        |                          |
 | [AJamez](https://osu.ppy.sh/users/7890134)            | Filipino                 |
 | [Benny-](https://osu.ppy.sh/users/4023183)            | Norwegian                |
-| [Electoz](https://osu.ppy.sh/users/6485263)           | Thai                     |
-| [JeirYagtama](https://osu.ppy.sh/users/7483452)       | Filipino                 |
+| [chickenbible](https://osu.ppy.sh/users/2306637) |  |
+| [IDu5t](https://osu.ppy.sh/users/6053071) | German |
 | [-Luminate](https://osu.ppy.sh/users/4778689)         | Thai                     |
```
Comment (O1CDeif) [inline wiki/People/Beatmap_Nominators/en.md]: -Luminate should go to the top of the list due to the hyphen

File: wiki/People/Beatmap_Nominators/en.md [hunk 2]
```diff
@@ -72,73 +72,90 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
 | [Hailie](https://osu.ppy.sh/users/5226970)            |                          |
+| [Hanasaka Yui](https://osu.ppy.sh/users/6443117) |  |
+| [Hinsvar](https://osu.ppy.sh/users/1249323) | Indonesian |
 | [hypercyte](https://osu.ppy.sh/users/9155377)         |                          |
 | [IceKalt](https://osu.ppy.sh/users/5410645)           | German                   |
-| [iYiyo](https://osu.ppy.sh/users/3919785)             | Spanish, some German     |
 | [jonathanlfj](https://osu.ppy.sh/users/270377)        | Chinese, some French     |
 | [Kaitjuh](https://osu.ppy.sh/users/2225327)           | Dutch                    |
 | [Kalibe](https://osu.ppy.sh/users/3376777)            | Polish                   |
+| [Krfawy](https://osu.ppy.sh/users/1478292) |  |
```
Comment (O1CDeif) [inline wiki/People/Beatmap_Nominators/en.md]: Additional language for Krfawy: Polish

File: wiki/People/Beatmap_Nominators/en.md [hunk 3]
```diff
@@ -72,73 +72,90 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
 | [TheKingHenry](https://osu.ppy.sh/users/5128277)      | Finnish                  |
 | [timemon](https://osu.ppy.sh/users/2072005)           | Thai                     |
-| [Trynna](https://osu.ppy.sh/users/2652951)            | Portuguese               |
 | [Ultima Fox](https://osu.ppy.sh/users/3198109)        |                          |
 | [UndeadCapulet](https://osu.ppy.sh/users/2523533)     |                          |
 | [Uta](https://osu.ppy.sh/users/9439796)               | Indonesian               |
+| [Venix](https://osu.ppy.sh/users/5999631) | Polish |
 | [Yahuri](https://osu.ppy.sh/users/4682629)            |                          |
 | [yaspo](https://osu.ppy.sh/users/4945926)             |                          |
 | [Yauxo](https://osu.ppy.sh/users/898306)              | German                   |
```
Comment (O1CDeif) [inline wiki/People/Beatmap_Nominators/en.md]: Yauxo retired

File: wiki/People/Beatmap_Nominators/en.md [hunk 4]
```diff
@@ -170,9 +187,10 @@ _Please note: All BN speak English unless otherwise noted._ Link to the [user gr
...
 | [Dellvangel](https://osu.ppy.sh/users/5186244)        | Indonesian               |
 | [DoNotMess](https://osu.ppy.sh/users/1596318)         | Indonesian               |
+| [K a b i -](https://osu.ppy.sh/users/6209713) | Chinese |
 | [Kawawa](https://osu.ppy.sh/users/4647754)            | Korean                   |
+| [KeppSado](https://osu.ppy.sh/users/11085809) |  |
+| [Leniane](https://osu.ppy.sh/users/7138602) | Dutch |
 | [MEGAtive](https://osu.ppy.sh/users/3094101)          | Indonesian               |
 | [Rivals_7](https://osu.ppy.sh/users/4610379)          | Indonesian               |
-| [Sandalphon](https://osu.ppy.sh/users/2633753)        | Chinese                  |
 | [Sinnoh](https://osu.ppy.sh/users/4236057)            |                          |
```
Comment (O1CDeif) [inline wiki/People/Beatmap_Nominators/en.md]: Sinnoh didn't pass probation for mania, should be removed from here

File: wiki/People/Beatmap_Nominators/en.md (hunk 2 above)
Comment (celerih) [reply wiki/People/Beatmap_Nominators/en.md]: done on all 4 languages to all your suggestions

**Comment by WalterToro** [comment]
Can you please clean up tables and remove extra spaces, pplease?

**Pull request merged**